Packs/CofenseTriage/README.md

```diff
@@ -14,4 +14,35 @@ The integration allows Cortex XSOAR to ingest phishing threats from Cofense Tria
 
 Configuration with Cofense Triage allows for continuous API polling for new threats located in the inbox or processed location queues. Analysts can setup playbooks to respond to security incidents created in Cortex XSOAR.
 
+### Playbooks
+
+Users would be able to run the playbook in 5.5.0 or higher version as it requires commands to execute the task.

```

Comment:
```suggestion
Users are only able to run the playbook in v6.0.0 or higher as it requires commands to execute the task.

```

Reply:
Updated the version note.
We have provided numbers instead of bullets to specify the list of playbooks.